### PR TITLE
chore: dockerize

### PR DESCRIPTION
**Pull request by kirisaki**
# Overview

- [x] Dockerize
  - [x] WebSocket
  - [x] API
  - [x] UI
  - [ ] ~~Worker~~
- [x] Bundle `docker-compose.yml`

## Summary by CodeRabbit

- **New Features**
	- Updated Dockerfiles for the API and UI components, enhancing build and deployment processes.
	- Simplified Node.js application deployment by removing multi-stage build steps and directly starting the app.
  
- **Bug Fixes**
	- None reported.

- **Refactor**
	- Restructured Dockerfile paths and working directories to improve organization and modularity.
	- Implemented a multi-stage build process for the WebSocket server, optimizing performance and security.

### CODE REVIEW

File: worker/Dockerfile [hunk 1]
```diff
@@ -1,4 +1,3 @@
-# syntax=docker/dockerfile:1
```
Comment (miseyu) [inline worker/Dockerfile:1]: @kirisaki 
Please don't delete things that are not necessary!

Comment (kirisaki) [reply worker/Dockerfile:1]: What do you mean?

Comment (pyshx) [reply worker/Dockerfile:1]: @kirisaki  Any reason why this line was deleted? Please retain the parser directive on at the beginning of the Dockerfile.

Comment (kirisaki) [reply worker/Dockerfile:1]: I deleted it because VS Code pointed out it. I revert it.

Comment (kirisaki) [reply worker/Dockerfile]: Done.

Comment (miseyu) [reply worker/Dockerfile]: Thank you so much 🙇

**Pull request closed without merging**